install/configuring_inventory_file.adoc

```diff
@@ -225,8 +225,7 @@ validity for etcd CA, peer, server and client certificates. Defaults to `1825`
 (5 years).
 
 |`openshift_certificate_expiry_warning_days`
-|The amount of time the auto-generated certificates must be valid for an upgrade
-to proceed. Defaults to `365` (1 year).
+|Flag certificates that will expire in this many days from now. Defaults to `365` (1 year).
```

Comment:
I think the original meaning has changed here, and I'm not sure it's clear enough what this flagging does. I think it should start "The amount of time the certificates must be valid for an upgrade to proceed."

Reply:
@kalexand-rh @maxwelldb my understanding of this option is that it is used by the certificate expiry playbook to flag a certificate for expiration if it is due to expire within the given amount of days. I'm not sure what the upgrade text is referring to, maybe the upgrade process does some kind of check for it?
cc @vrutkovs @sdodson might be able to clarify.